omod/src/main/resources/config.xml

```diff
@@ -27,27 +27,31 @@
 			org.openmrs.module.uilibrary
 		</require_module>
 
-		<require_module version="1.0.2">
+		<require_module version="1.0.0">
```

Comment:
You can inherit the version from the main pom file as global variables as done here https://github.com/uonafya/ehrreports/blob/master/omod/src/main/resources/config.xml#L28-L39, those variables are declared in the main pom file like this https://github.com/uonafya/ehrreports/blob/master/pom.xml#L17-L26

Reply:
I am trying to inherit as requested but im getting this error from main pom. 
'dependencies.dependency.version' for org.openmrs.module:uilibrary-api:jar must be a valid version but is '${uilibraryVersion}'.